src/main/java/seedu/address/model/team/TeamList.java

```diff
@@ -0,0 +1,25 @@
+package seedu.address.model.team;
+
+import java.util.ArrayList;
+
+import seedu.address.model.person.Person;
+
+public class TeamList {
+    private ArrayList<Person> list;
+
+    TeamList() {
```

Comment:
Here as well

Reply:
Hi Lin Da, I have added the access modifier as well as the java doc comments. Thanks for pointing it out!!